default/scripting/techs/growth/CYBORG.focs.txt

```diff
@@ -24,10 +24,9 @@ Tech
             ]
             stackinggroup = "CYBORG_TROOPS_STACK"
             priority = [[VERY_EARLY_PRIORITY]]
-            effects = SetMaxTroops value = Value + Target.Population * [[TROOPS_PER_POP]] accountinglabel = "GRO_CYBORG"
+            effects = SetMaxTroops value = Value + 1.5 * Target.Population * [[TROOPS_PER_POP]] accountinglabel = "GRO_CYBORG"
```

Comment:
I think you should use a named value ref and use that in the stringtable
`value = Value + Target.Population * NamedReal name = "CYBORG_TROOPS_EFFECT" value = 1.5 * [[TROOPS_PER_POP]] accountinglabel = "GRO_CYBORG" `

also add to stringtable
`CYBORG_TROOPS_EFFECT
per population`

Reply:
What's the reason to add
'''
CYBORG_TROOPS_EFFECT
per population
'''
?

Reply:
Currently the named value ref s need an a stringtable entry in order to work. I suggest you try my suggestion and come back with UI feedback

Reply:
Now I got it (after testing properly). I need to change some of the new stringtable entries from `''''''` to `per population` and this will be solved.